doc/classes/ProjectSettings.xml

```diff
@@ -2699,17 +2699,19 @@
 			[/codeblock]
 		</member>
 		<member name="threading/worker_pool/low_priority_thread_ratio" type="float" setter="" getter="" default="0.3">
+			If [member threading/worker_pool/use_system_threads_for_low_priority_tasks] is [code]false[/code], this value determines the ratio of low-priority tasks that can be executed at a time by [WorkerThreadPool]'s threads. For example, if 10 threads are available and this value is set to [code]0.3[/code], at most 3 low-priority tasks can be executed by the [WorkerThreadPool] at a time.
 		</member>
 		<member name="threading/worker_pool/max_threads" type="int" setter="" getter="" default="-1">
 			Maximum number of threads to be used by [WorkerThreadPool]. Value of [code]-1[/code] means no limit.
 		</member>
 		<member name="threading/worker_pool/use_system_threads_for_low_priority_tasks" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], [WorkerThreadPool]'s threads will be reserved for high-priority tasks.
```

Comment:
The actual meaning is this:
If true, low priority tasks won't run on the threads from the pool. Instead, for each low priority task a new thread will be created and finally destroyed when the task is over. Ratio of low-prio tasks still apply.